lib/pubsub.rb

```diff
@@ -11,6 +11,12 @@ def topic(name)
     client.topic(name) || client.create_topic(name)
   end
 
+  def subscription(name)
+    client.subscription(name) || topic(name).subscribe(name,
+      dead_letter_topic: topic('morgue'),
+      dead_letter_max_delivery_attempts: 2)
```

Comment:
What will this do?

Reply:
Yes, but what is the consequence for our job processing?

Reply:
If the job fails, it should be enqueued to the morgue queue after two attempts.